### PR TITLE
fix: Allow users to disable ANSI color in logging

### LINKED ISSUE #477: Allow users to disable ANSI color in logging
**Is your feature request related to a problem? Please describe.**
Botkube logs seem to have [ANSI color codes on by default](https://github.com/infracloudio/botkube/blob/develop/pkg/log/log.go#L42) (and provide no configuration to disable them). In our cluster, we ship logs to an ELK stack offsite (and do not often view them using a traditional terminal). The ANSI codes impair readability of logs when not viewed in a terminal.

**Describe the solution you'd like**
Provide a setting or configuration that allows users to disable ANSI escape codes in log output.

**Describe alternatives you've considered**
n/a

**Additional context**
Here's an example log line I see in our Kibana instance:

```
[36mINFO[0m[2021-04-26T15:52:21Z] Registering the filter IngressValidator      
```

### PR DESCRIPTION
**Pull request by sumit-tembe**
##### ISSUE TYPE
 - Bug fix Pull Request

Fixes #477

### CODE REVIEW

File: pkg/log/log.go [hunk 1]
```diff
@@ -39,7 +40,8 @@ func init() {
 		logLevel = logrus.InfoLevel
 	}
 	log.SetLevel(logLevel)
-	log.Formatter = &logrus.TextFormatter{ForceColors: true, FullTimestamp: true}
+	forceColors, _ := strconv.ParseBool(os.Getenv("LOG_FORCE_COLORS"))
```
Comment (PrasadG193) [inline pkg/log/log.go:43]: what value do we get when `ParseBool` fails? We should check the error and set forceColors=false if is not nil

Comment (sumit-tembe) [reply pkg/log/log.go:43]: ParseBool returns `false` as default value. (Even in case of an error)

Comment (PrasadG193) [reply pkg/log/log.go:43]: I still think it's better to handle error for readability

**Comment by pkosiec** [comment]
Hi @sumit-tembe, cool PR - would you like to reopen and rebase it to latest `develop`? It should be even easier to do than before, as now we are using `envconfig`: https://github.com/infracloudio/botkube/blob/develop/cmd/botkube/main.go#L48

Thanks!

**Pull request closed without merging**